content/03.aim1.md

```diff
@@ -46,20 +48,79 @@ Simulation statistics under different parameters:
+
 Clustering performance was measured by normalized mutual information (NMI).
 NMI is an normalization of the Mutual Information (MI, measures the dependence of two random variables). It is a measurement to determine the quality of clustering, which is between 0 (no mutual information) and 1 (perfect correlation).
 
 ![](images/nmi.png)
 
-Figure 3: Performance comparison among VAE, t-SNE and PCA under different simulation parameters. 
+Figure 3: Performance comparison among VAE, t-SNE and PCA under different simulation parameters.
+
+#### VAE performance on real single cell datasets
+
+##### 1. real single cell datasets:
+
+To evaluate the performance of VAE and the other dimension reduction approaches, we select 8 real single cell datasets with true celltype labels.
+
+| dataset | ncells | ngenes | tissue | organism | celltypes | Accesion number | Platform | celltype annotation |
+| ------- | ------ | ------ | ------ | -------- | --------- | --------- | ------------------ | -------------------------------------- |
+| yan | 90 | 20214 | Embryo Devel | human | 6 | GSE36552 | Illumina HiSeq 2000 | celltypes derived from different embryonic developmental time points |
+| goolam | 124 | 41428 | Embryo Devel | mouse | 5 | E-MTAB-3321 | | 5 celltypes derived from different embryonic developmental time points |
+| pollen | 301 | 23730 | Developing Cerebral Cortex | human | 11 | SRP041736 | Illumina HiSeq 2000 | 11 celltypes defined by different celllines |
+| patel | 430 | 5948 | Glioblastoma | human | 12 | GSE57249 | Illumina HiSeq 2000 | 5 celltypes defined from five primary glioblastomas |
+| nestorowa | 1656 | 4773 | Glioblastoma | human | 12 | GSE81682 | Illumina HiSeq 2500 | 9 celltypes defined by sorting |
+| Petropoulos | 1529 | 26178 | Embryo Devel | human | 6 | E-MTAB-3929 | Illumina HiSeq 2000 | celltypes defined by developmental stages |
+| HCA melanoma data | 6639 | 22489 | Lymph node | mouse | 9 | HCA release | Smart-seq2 |
+| baron-human | 8569 | 20125 | Pancreas | human | 14 | GSE84133 | Illumina HiSeq 2500 | 14 celltypes defined based on expression profiles and cell origin |
+
+##### 2. Visualization of real single cell data based on differnt structure of VAE: VAE_depth3 (3 hidden layer), VAE_depth2 (2 hidden layer), VAE_depth1 (1 hidden layer), t-SNE, ZIFA, umap and PCA
+
+We tested the performance of VAE and another 4 methods (t-SNE, ZIFA, umap and PCA) on these real datasets with different number of cells, genes and celltypes. 
+Figure 4 shows the 2-D visulization of the real single cell datasets. 
+PCA and ZIFA generally performs better when number of cells is small, but with the increase of number of cells, VAE shows a good power to differentiate different celltypes.
+
+![](images/real.data.vis.png)
+
+Figure 4: 2D Visualization of real single cell data using VAE_depth3 (3 hidden layer), VAE_depth2 (2 hidden layer), VAE_depth1 (1 hidden layer), t-SNE, ZIFA, umap and PCA.
+
```

Comment:
Are there any consistent notable differences between the VAEs of depth 1, 2, and 3 that you want to comment on above? I suspect that could also be of interest to readers.